api/src/main/java/org/openmrs/api/db/hibernate/NativeIfNotAssignedIdentityGenerator.java

```diff
@@ -19,7 +19,7 @@
 import org.hibernate.HibernateException;
 import org.hibernate.MappingException;
 import org.hibernate.dialect.Dialect;
-import org.hibernate.engine.SessionImplementor;
+import org.hibernate.engine.spi.SessionImplementor;
 import org.hibernate.id.Configurable;
 import org.hibernate.id.IdentityGenerator;
 import org.hibernate.persister.entity.EntityPersister;

```

Comment:
@harsha89  @djazayeri  isn't this the custom id generator that will no longer be needed since we are using the standard hibernate jar file, i see it is being used in the mapping files for concepts and active list items